build.gradle

```diff
@@ -1,3 +1,12 @@
+buildscript {
+    configurations.all {
+        resolutionStrategy {
+            force("org.ow2.asm:asm:9.3")
+            force("org.ow2.asm:asm-commons:9.3")
+        }
+    }
+}
+
```

Comment:
This is mandatory to force those versions because it would not remapJar successfully (based on https://github.com/pop4959/Chunky/commit/b5d4060fbd87e3263d6051aa4193fdf43acf6074)